src/main/java/br/com/quintoandar/javajwt/QuintoAndarKeycloakJwtBean.java

```diff
@@ -0,0 +1,89 @@
+package br.com.quintoandar.javajwt;
+
+import org.jose4j.jwk.RsaJsonWebKey;
+import org.jose4j.jwt.consumer.InvalidJwtException;
+import org.jose4j.jwt.consumer.JwtConsumer;
+import org.jose4j.jwt.consumer.JwtConsumerBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.IOException;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.interfaces.RSAPublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.Base64;
+import java.util.Map;
+import java.util.Optional;
+
+public class QuintoAndarKeycloakJwtBean implements QuintoAndarKeycloakJwt {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(QuintoAndarKeycloakJwtBean.class);
+
+    private static final String KEY_ALGORITHM = "RSA";
+
+    private QuintoAndarKeycloakPublicKeyService quintoAndarKeycloakPublicKeyService;
+
+    private JwtConsumer jwtConsumer;
+
+    @Autowired
+    public QuintoAndarKeycloakJwtBean(final QuintoAndarKeycloakPublicKeyService quintoAndarKeycloakPublicKeyService) {
+        this.quintoAndarKeycloakPublicKeyService = quintoAndarKeycloakPublicKeyService;
+    }
+
+    @Autowired
```

Comment:
is this `@Autowired` necessary? it's not a constructor nor a setter

Reply:
Not sure if is necessary, I followed the idea in `QuintoAndarJwtBean` and since I don't know much about making packages in Java I assumed it was something required to export the info to whatever imported this package :thinking: I can remove it though :thinking:

Reply:
Actually, my bad! It was supposed to be @Override, thank you :facepalm:

Reply:
Done!